go/apis/kube-mail/v1alpha1/emailpolicy_types.go

```diff
@@ -0,0 +1,56 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type EmailPolicySMTPSink struct {
+	Server      v1.ObjectReference `json:"server"`
+	Credentials v1.ObjectReference `json:"credentials"`
+}
+
+type EmailPolicyRateLimiting struct {
+	Maximum int    `json:"maximum"`
+	Period  string `json:"period"`
+}
+
+type EmailPolicySink struct {
+	SMTP EmailPolicySMTPSink `json:"smtp"`
+}
+
+type EmailPolicySpec struct {
+	Default      bool                    `json:"default"`
+	PodSelector  metav1.LabelSelector    `json:"podSelector"`
+	RateLimiting EmailPolicyRateLimiting `json:"ratelimiting"`
```

Comment:
All these are optional in the original spec:

https://github.com/mittwald/kube-mail/blob/6f9d3d3978be3e309a13edb60493ab2383594f07/src/k8s/types/v1alpha1/emailpolicy.ts#L15-L23

Reply:
done